test/common/upstream/upstream_impl_test.cc

```diff
@@ -1785,6 +1806,59 @@ TEST_F(ClusterInfoImplTest, ExtensionProtocolOptionsForFilterWithOptions) {
   }
 }
 
+TEST_F(ClusterInfoImplTest, TestNoSocketOptionsDefault) {
+  auto cluster = makeCluster(strictDnsClusterWithAttrJson(""));
```

Comment:
You'll need to convert thse to use parseClusterFromV2Yaml. Not sure it's worth introducing helpers vs just inlining the cluster config.

Reply:
I'll convert them. I prefer to keep this sort of stuff helpers, simply because, IMHO, it improves the readability of the test -- less lines to parse to get to the good stuff. Though, I guess in this case the fact that there are no options isn't really explicit... A comment to the args (i.e. /* no attributes means no socket options */) could make that more clear.